hack/ginkgo-e2e.sh

```diff
@@ -169,15 +169,28 @@ elif [[ "${E2E_TEST_DEBUG_TOOL}" == "gdb" ]]; then
   program=("gdb")
 fi
 
+debug_opt=()
+# Here are some options which will be helpful for debugging, for example, test hangs for some reason (https://github.com/kubernetes/kubernetes/issues/111086).
+
+# Unmask this line to circumvent cases where the code grabbing the stdout/stderr pipe
+# is not under the framework control and may cause hangs (https://github.com/onsi/ginkgo/issues/851)
+# this option will also help to reveal more information when the test hangs (https://github.com/onsi/ginkgo/issues/970).
+# debug_opt+=(--ginkgo.output-interceptor-mode=none)
+
+# Unmask this line in order to emits more output including GinkgoWriter contents.
+# debug_opt+=(--ginkgo.v)
+
+# Unmask this line if you want the default reporter prints out captured output of passed tests.
+# debug_opt+=(--ginkgo.always-emit-ginkgo-writer)
```

Comment:
Same here: does this have an effect, considering that we prune the JUnit output?

Reply:
it won't I think, as it's set to empty if the testcase passed, will remove this one.